environments/swiss/public.yml

```
@@ -126,6 +126,8 @@ localsettings:
   REMINDERS_QUEUE_ENABLED: False
   SMS_GATEWAY_URL: 'http://gw1.promessaging.com/sms.php'
   SMS_QUEUE_ENABLED: False
+  SES_CONFIGURATION_SET: swiss-email-events
+  SNS_EMAIL_EVENT_SECRET: "{{ localsettings_private.SNS_EMAIL_EVENT_SECRET }}"
```

oops, actually, for work I'm doing on the secrets backends, I think this would have to be changed to:

```suggestion
  SNS_EMAIL_EVENT_SECRET: "{{ SNS_EMAIL_EVENT_SECRET }}"
```

---

done! thanks